stonesoup/plotter.py

```diff
@@ -271,6 +274,51 @@ def plot_tracks(self, tracks, mapping, uncertainty=False, particle=False, track_
         else:
             self.ax.legend(handles=self.handles_list, labels=self.labels_list)
 
+    def plot_density(self, state_sequences: Iterable[StateMutableSequence],
```

Comment:
It'd be cool to extend this (maybe raise as a separate issue) to support Gaussian States and Gaussian Mixture States